CONTRIBUTING.md

```diff
@@ -124,7 +124,7 @@ This is the same set of tests that is run on our CI system, CircleCI.
 If your patch makes a change that would be visible or interesting to a
 user of SourceCred—for example, fixing a bug—please add a description of
 the change under the `[Unreleased]` heading of `CHANGELOG.md`. Your new
-change should be the first entry in the section.
+change should be the first entry in the section. The format of your entry should be: ```<description of change> (#<PR number>)```.
```

Comment:
Nit (here and elsewhere): could we keep this document wrapped consistently? (I believe it's 80 chars)

Reply:
Yes, please, but it’s wrapped at 72 characters, not 80.

Reply:
How do you all generally enforce this? Is there something in place that will do this for all files or should I be manually doing this using something like `fold`?

Reply:
It would be great to have some tooling that does this automatically. (Like prettier but for Markdown?) Personally, I code/write in vim, and re-wrapping a paragraph is easy as `gqap`